{
    "test_ipfs_accelerate": {
        "test_ipfs_accelerate": "Error initializing worker:\nException type: AttributeError\nException message: 'OSError' object has no attribute 'keys'\nTraceback:\nTraceback (most recent call last):\n  File \"C:\\Users\\barberb\\ipfs_accelerate_py\\ipfs_accelerate_py\\test\\test_ipfs_accelerate.py\", line 74, in test\n    ipfs_accelerate_init = await self.ipfs_accelerate_py.init_endpoints( [model], resources)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\barberb\\ipfs_accelerate_py\\ipfs_accelerate_py\\ipfs_accelerate.py\", line 396, in init_endpoints\n    resource_list = list(self.worker_resources.keys())\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^\nAttributeError: 'OSError' object has no attribute 'keys'\n",
        "test_backend": "Error initializing worker:\nException type: AttributeError\nException message: 'test_backend_py' object has no attribute '__test__'\nTraceback:\nTraceback (most recent call last):\n  File \"C:\\Users\\barberb\\ipfs_accelerate_py\\ipfs_accelerate_py\\test\\test_ipfs_accelerate.py\", line 88, in test\n    test_results[\"test_backend\"] = self.test_backend.__test__(resources, metadata)\n                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^\nAttributeError: 'test_backend_py' object has no attribute '__test__'. Did you mean: '__dict__'?\n"
    }
}